LibCore/CProcess.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <AK/AKString.h>
+
+class CProcess {
+public:
+    ~CProcess();
+
+    enum Priority {
+        Highest,
+        Normal,
+        Lowest,
+    };
+
+    static CProcess* start_detached(const StringView& program, const Vector<StringView>& arguments = Vector<StringView>(), Priority = Priority::Normal);
```

Comment:
We should return the `CProcess` wrapped in some sort of smart pointer.
`OwnPtr<CProcess>` is probably the most suitable.
Also you can write `const Vector<StringView>& arguments = {}` for a bit less redundant redundancy :)